manim/mobject/geometry.py

```diff
@@ -281,7 +281,7 @@ def __init__(
 
     def generate_points(self):
         self.set_pre_positioned_points()
-        self.scale(self.radius, about_point=ORIGIN)
+        self.scale(self.radius, about_point=ORIGIN)  # similar to here?
```

Comment:
This comment is only temporary and to reference how the radius is set in the first place.